### ISSUE TITLE
Build doesn't pass when trying to compile current website version

### ISSUE THREAD

**Issue by vindex10** [Lyndir/MasterPassword#235]
I'm receiving the following error:
```
/usr/lib/gcc/x86_64-pc-linux-gnu/6.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: /tmp/ccZeScFv.o: undefined reference to symbol 'tgetstr'
/lib64/libtinfo.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

To fix it, it's enough to run `./build` setting `LDFLAGS=-ltinfo` explicitly:
```bash
LDFLAGS=-ltinfo ./build
```

**Comment by lhunath** [comment]
Is this off `master`?

**Comment by vindex10** [comment]
On `master` it doesn't happen. But still, I think it could be useful for those who build version from web page.

**Comment by lhunath** [comment]
Should be fixed now in `mpw-2.6-cli-5-0-g344771db`.
